src/status_im/contexts/wallet/add_account/add_address_to_save/view.cljs

```diff
@@ -0,0 +1,82 @@
+(ns status-im.contexts.wallet.add-account.add-address-to-save.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.common.floating-button-page.view :as floating-button-page]
+    [status-im.contexts.wallet.add-account.add-address-to-save.style :as style]
+    [status-im.contexts.wallet.common.activity-indicator.view :as activity-indicator]
+    [status-im.contexts.wallet.common.address-input.view :as address-input]
+    [status-im.contexts.wallet.utils :as wallet.utils]
+    [status-im.subs.wallet.add-account.address-to-watch]
+    [utils.ens.core :as utils.ens]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- on-press-confirm-add-address-to-save
+  [input-value]
+  (rf/dispatch
+   [:wallet/confirm-add-address-to-save
+    {:address input-value
+     :ens?    (utils.ens/is-valid-eth-name?
+               input-value)}]))
+
+(defn view
+  []
+  (let [addresses (rf/sub [:wallet/lowercased-saved-addresses])
```

Comment:
A new sub to use lowercased addresses, Because when there's case changes in addresses text duplication is hard to detect.